lib/Home.dart

```diff
@@ -10,7 +10,7 @@ class Greeting extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      child: Text('Hi $name!', style: Theme.of(context).textTheme.headline),
+      child: Text('Hi $name!', style: Theme.of(context).textTheme.headline5),
```

Comment:
Define both headline5 and subtitle1 in the TextTheme if you are going to use them. Otherwise, the build on GitHub will keep failing.

Reply:
I'm not really sure what the TextTheme is and what the GitHub build is for?

Reply:
Names of the headline5 might have changed according to https://api.flutter.dev/flutter/material/TextTheme-class.html

Reply:
@helenxuyang A TextTheme defines typography for your entire app. You can find it in the _buildApp function in main_common.dart. I would just change the definition for headline to headline5. And I would change the definition for subhead to subtitle1. For reference, you can look at the second brown box on the page that same provided in the above comment.